addons/l10n_cz/i18n/l10n_cz.pot

```diff
@@ -56,7 +56,12 @@ msgstr ""
 #: model_terms:ir.ui.view,arch_db:l10n_cz.l10n_cz_external_layout_standard
 #: model_terms:ir.ui.view,arch_db:l10n_cz.l10n_cz_external_layout_striped
 #: model_terms:ir.ui.view,arch_db:l10n_cz.report_invoice_document
-msgid "ID:"
+msgid "Company ID:"
+msgstr ""
+
+#. module: l10n_cz
+#: model:ir.model.fields,field_description:l10n_cz.field_base_document_layout__account_fiscal_country_id
+msgid "Fiscal Country"
```

Comment:
this term is already in the file, before "Company ID:"
Maybe we should move the added term just after "Company ID" (same for cs.po, which also has the translation already)

Reply:
My bad, went a bit too fast and didn't noticed it broke alphabetical order, it's fixed :flushed: